hawkular-alerts-rest-tests/src/test/resources/standalone-ha.xsl

```diff
@@ -67,6 +67,9 @@
       <replicated-cache name="publish" mode="ASYNC">
         <transaction mode="BATCH"/>
       </replicated-cache>
+      <replicated-cache name="dataIds" mode="ASYNC">
+        <transaction mode="BATCH"/>
+      </replicated-cache>
       <replicated-cache name="schema" mode="SYNC">
```

Comment:
The publish and dataIds caches are replicated and that makes sense because it is unknown which node will process a definitions update (right?).   But I think as it is written the CacheManager will perform the initialInitialization on every node.  That does not seem correct, should it be limited to the coordinator?

I suppose an alternative would be to make these local caches but to propogate DefinitionEvents across the cluster.